README.md

```diff
@@ -17,35 +17,43 @@ Install dependencies
 yarn
 ```
 
-and then run the build:
```

Comment:
There is a difference between running a component and running the website.
Let's keep them separate please